lib/notification-issue.coffee

```diff
@@ -52,7 +54,10 @@ class NotificationIssue
       "#{repoUrl}/issues/new?title=#{@encodeURI(@getIssueTitle())}&body=#{@encodeURI(issueBody)}"
 
   getIssueTitle: ->
-    @notification.getMessage()
+    title = @notification.getMessage()
+    if title.length > TRUNCATE_TITLE_TO
```

Comment:
We only want to add the ... ellipsis if the length is bigger than 100 characters, right?


> On 03/05/2015, at 17:44, Lee Dohm <notifications@github.com> wrote:
> 
> In lib/notification-issue.coffee:
> 
> > @@ -52,7 +54,10 @@ class NotificationIssue
> >        "#{repoUrl}/issues/new?title=#{@encodeURI(@getIssueTitle())}&body=#{@encodeURI(issueBody)}"
> >  
> >    getIssueTitle: ->
> > -    @notification.getMessage()
> > +    title = @notification.getMessage()
> > +    if title.length > TRUNCATE_TITLE_TO
> It looks like checking the length isn't necessary:
> 
> foo = '123'    // => "123"
> foo = foo.substring(0, 10) + '...'     // => "123..."
> So 58-60 can just be compressed into:
> 
> title.substring(0, TRUNCATE_TITLE_TO - 3) + '...'
> —
> Reply to this email directly or view it on GitHub.
>